modules/lnd/lnd_wrapper/wrapper.go

```diff
@@ -162,6 +162,31 @@ func LndParseP2pLightningMessage(data *C.char, length C.int) *C.char {
 	case 19:
 		sb.WriteString("MSG_TYPE=pong;IGNORED=")
 		sb.WriteString(fmt.Sprintf("%d", len(message.(*lnwire.Pong).PongBytes)))
+	case 34:
```

Comment:
There's a lot of casting to `*FundingCreated` below.  We could improve readability by doing the cast one time  here.
```suggestion
	case 34:
		fc := message.(*lnwire.FundingCreated)
```

Reply:
Yes. Fixed